wire-ios/Wire-iOS/Sources/AppRootRouter.swift

```diff
@@ -250,9 +250,52 @@ extension AppRootRouter: AppStateCalculatorDelegate {
         case let .locked(userSession):
             screenCurtainWindow.userSession = userSession
             showAppLock(userSession: userSession, completion: completion)
+        case let .syncFailure(error, onRetry):
+            presentSyncErrorAlert(error: error, onRetry: onRetry)
         }
     }
 
+    private func presentSyncErrorAlert(
+        error: any Error,
+        onRetry: @escaping () -> Void
+    ) {
+
+        let syncErrorMessage = if DeveloperFlag.showDetailedSyncError.isOn {
```

Comment:
suggestion: Use a more general flag. This is useful in many places such as the new login flow.
```suggestion
        let syncErrorMessage = if DeveloperFlag.showDetailedError.isOn {
```

Reply:
done in https://github.com/wireapp/wire-ios/pull/2954/commits/75278fa1a6794336c47765f5b8f0d95ff9c57b02